src/__tests__/tw.spec.ts

```diff
@@ -12,20 +12,34 @@ describe(`tw`, () => {
   beforeEach(() => (tw = create()));
 
   test(`font-sizes`, () => {
-    expect(tw`text-xs`).toMatchObject({ fontSize: 12 });
-    expect(tw`text-sm`).toMatchObject({ fontSize: 14 });
-    expect(tw`text-base`).toMatchObject({ fontSize: 16 });
-    expect(tw`text-lg`).toMatchObject({ fontSize: 18 });
-    expect(tw`text-2xl`).toMatchObject({ fontSize: 24 });
-    expect(tw`text-3xl`).toMatchObject({ fontSize: 30 });
-    expect(tw`text-4xl`).toMatchObject({ fontSize: 36 });
-    expect(tw`text-5xl`).toMatchObject({ fontSize: 48 });
-    expect(tw`text-6xl`).toMatchObject({ fontSize: 60 });
-    expect(tw`text-7xl`).toMatchObject({ fontSize: 72 });
-    expect(tw`text-8xl`).toMatchObject({ fontSize: 96 });
-    expect(tw`text-9xl`).toMatchObject({ fontSize: 128 });
+    expect(tw`text-xs`).toMatchObject({ fontSize: 12, lineHeight: 16 });
+    expect(tw`text-sm`).toMatchObject({ fontSize: 14, lineHeight: 20 });
+    expect(tw`text-base`).toMatchObject({ fontSize: 16, lineHeight: 24 });
+    expect(tw`text-lg`).toMatchObject({ fontSize: 18, lineHeight: 28 });
+    expect(tw`text-2xl`).toMatchObject({ fontSize: 24, lineHeight: 32 });
+    expect(tw`text-3xl`).toMatchObject({ fontSize: 30, lineHeight: 36 });
+    expect(tw`text-4xl`).toMatchObject({ fontSize: 36, lineHeight: 40 });
+    expect(tw`text-5xl`).toMatchObject({ fontSize: 48, lineHeight: 48 });
+    expect(tw`text-6xl`).toMatchObject({ fontSize: 60, lineHeight: 60 });
+    expect(tw`text-7xl`).toMatchObject({ fontSize: 72, lineHeight: 72 });
+    expect(tw`text-8xl`).toMatchObject({ fontSize: 96, lineHeight: 96 });
+    expect(tw`text-9xl`).toMatchObject({ fontSize: 128, lineHeight: 128 });
   });
 
+  test(`font-sizes with relative line-height`, () => {
+    const config: TwConfig = { 
+      theme: { 
+        fontSize: { 
+          relative: [`1.25rem`, { lineHeight: `1.5` }],
```

Comment:
if you configure the line height in the other way that tailwind supports (two strings) like this:

```ts
twostrings: [`1.25rem`, `1`]
```

Then you'll get a test failure if you do:

```ts
expect(tw`text-twostrings`).toMatchObject({ fontSize: 20, lineHeight: 20 });
```

Reply:
Thanks, completely missed this way of configuring tailwind.